task.sql

```diff
@@ -1,2 +1,4 @@
 # Write your SQL code for the database creation here. Good luck! 
 USE ShopDB; 
+CREATE INDEX index_email ON ShopDB.Customers(Email);
+CREATE INDEX index_product ON ShopDB.Products(Name);
```

Comment:
Rename indexes, according to task description 
```suggestion
CREATE INDEX Email ON ShopDB.Customers(Email);
CREATE INDEX Name ON ShopDB.Products(Name);
```